### PR TITLE
Mobile: do not use span inside of a button

### PR DESCRIPTION
**Pull request by pedropintosilva**
Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: I88f1b97939c59927686bd72ea295f87abffcbe29

### CODE REVIEW

File: loleaflet/src/control/Control.JSDialogBuilder.js [hunk 1]
```diff
@@ -1347,9 +1347,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			image.src = data.image;
```
Comment (eszkadev) [inline loleaflet/src/control/Control.JSDialogBuilder.js:1347]: this image inserted into pushbutton will be overwritten by innerHTML call later I think

**Review by eszkadev — CHANGES_REQUESTED** [review]

File: loleaflet/src/control/Control.JSDialogBuilder.js (hunk 1 above)
Comment (eszkadev) [reply loleaflet/src/control/Control.JSDialogBuilder.js:1347]: also please check if sidebar popups which have buttons with images are correctly shown

**Comment by pedropintosilva** [comment]
@Rash419 could you please take it over?

- I have fixed https://github.com/CollaboraOnline/online/issues/2812
- But now we need to make sure that we account for the cases Szymon described above

**Comment by Rash419** [comment]
> @Rash419 could you please take it over?
> 
> * I have fixed [Mobile: do not use span inside of a button element #2812](https://github.com/CollaboraOnline/online/issues/2812)
> * But now we need to make sure that we account for the cases Szymon described above
> 
> feel free to add commits to this PR

Yes syzmon is right if there will a button with image and text both the image will be replaced with text but I could find any instance where both is there in button @pedropintosilva

**Review by eszkadev — APPROVED** [review]
looks good

**Pull request merged**